### PR TITLE
Yasir's js file added

### PR DESCRIPTION
**Pull request by mdyasir1**
Yasir's JS file added

### CODE REVIEW

**Review by zakeer — CHANGES_REQUESTED** [review]

**Comment by mdyasir1** [comment]
var firstname = "Yasir";
var lastname = " Mohammed";

var fullname =firstname + lastname;

var location= " Vijayawada";
var email="mdyasir4145@gmail.com";

console.log(`${fullname} from ${location}`)
console.log(`My email is ${email}`)

File: courses/javascript/task/yasir.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+var firstname = "Yasir";
+var lastname = " Mohammed";
+
+var fullname =firstname + lastname;
+
+var location= " Vijayawada";
```
Comment (zakeer) [inline courses/javascript/task/yasir.js:6]:
```suggestion
var location= "Vijayawada";
```

Remove starting space

File: courses/javascript/task/yasir.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+var email="mdyasir4145@gmail.com";
+
+console.log(`${fullname} from${location}`)
```
Comment (zakeer) [inline courses/javascript/task/yasir.js:9]:
```suggestion
console.log(`${fullname} from ${location}`)
```

File: courses/javascript/task/yasir.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+var firstname = "Yasir";
+var lastname = " Mohammed";
```
Comment (zakeer) [inline courses/javascript/task/yasir.js:2]:
```suggestion
var lastname = "Mohammed";
```

File: courses/javascript/task/yasir.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+var fullname =firstname + lastname;
```
Comment (zakeer) [inline courses/javascript/task/yasir.js:4]:
```suggestion
var fullname = `${firstname} ${lastname}`;
```

**Review by zakeer — CHANGES_REQUESTED** [review]

**Pull request merged**